### PR TITLE
SCRIPTS: Quest - Gullible's Travels

### PR DESCRIPTION
**Pull request by The-Halcyon-Days**
Messed up my fork and branches causing and figured better to start clean, so just going to do a new pull request. 

-Implemented the changes mentioned and tested the quest once and it worked without issue.

### CODE REVIEW

File: scripts/zones/Kazham/npcs/Magriffon.lua [hunk 1]
```diff
@@ -1,25 +1,50 @@
...
 -----------------------------------
 -- onTrigger Action
 -----------------------------------
 
-function onTrigger(player,npc)
-	player:startEvent(0x008F);
+function onTrigger(player,npc)   
+    local gulliblesTravelsStatus = player:getQuestStatus(OUTLANDS, GULLIBLES_TRAVELS);
+    local magriffonGilRequest = gulliblesTravelsStatus == QUEST_ACCEPTED --Checks if Gullible's Travels is started and gets previously requested gil #
+    and player:getVar("MAGRIFFON_GIL_REQUEST");
```
Comment (TeoTwawki) [inline scripts/zones/Kazham/npcs/Magriffon.lua]: This "and" is orphaned. Can't do that.

**Comment by xdemolish** [comment]
squash and force push (your latest commit doesn't have the SCRIPTS label)

**Comment by The-Halcyon-Days** [comment]
I have no idea how to git

**Comment by The-Halcyon-Days** [comment]
I hate git, remaking

**Pull request closed without merging**